app/components/accordion_sections/site_map_component.rb

```diff
@@ -17,7 +17,8 @@ def change_request_link_path
         )
       else
         new_planning_application_validation_red_line_boundary_change_validation_request_path(
-          planning_application
+          planning_application,
+          type: "red_line_boundary_change"
```

Comment:
How come this needs the type of validation request both in the method name and as a parameter? Can we avoid that somehow?